packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -452,6 +480,22 @@ describe("get repository, own shared and peer shared attributes", () => {
             const repositoryAttributes = result.value;
             expect(repositoryAttributes).toStrictEqual([services1RepoSurnameV0, services1RepoSurnameV1, services1RepoGivenNameV0, services1RepoGivenNameV1]);
         });
+
+        test("should allow to get only default attributes", async function () {
```

Comment:
Why is this test duplicated for the `GetAttributes` function, but the test `"should allow not to get default attributes"` is done for the `GetAttributes` function, but not for the `GetRepositoryAttributes` function?

Reply:
Testing the negative is not really something that is connected to this PR, but to the query itself. I was wondering how it would work so I added the test above. But I don't think it is necessary to add such a test everywhere a query is used (as can be seen that there wasn't such a test before).